underfs/hdfs/src/main/java/alluxio/underfs/hdfs/activesync/SupportedHdfsActiveSyncProvider.java

```diff
@@ -286,6 +282,7 @@ public void pollEvent(DFSInotifyEventInputStream eventStream) {
               count + currentlyBehind - behind ,
               end - start,
               String.format("%.2f", (count + currentlyBehind - behind) * 1000.0 / (end - start)));
+          LOG.info("Currently TxidsBehindEstimate by {}", currentlyBehind);
```

Comment:
Do you mind making the sequence of logging `HDFS Generated`, `# Processed`, `# Behind`?